data/hlint.yaml

```diff
@@ -264,7 +263,7 @@
 
     # FOLDS
 
-    - warn: {lhs: foldr  (>>) (return ()), rhs: sequence_}
+    - warn: {lhs: foldr  (>>) (return ()), rhs: sequenceA_}
```

Comment:
`>>` induces a Monad constraint, so sequence_ seems the more correct thing to suggest.